content/en/guide/v10/typescript.md

```diff
@@ -258,72 +331,7 @@ const AppContext = createContext({
 // }>
 ```
 
-It also requires you to pass in all the properties you defined in the initial value:
-
-```tsx
-function App() {
-  // This one errors 💥 as we haven't defined theme
-  return (
-    <AppContext.Provider
-      value={{
-//    ~~~~~ 
-// 💥 Error: theme not defined
-        lang: "de",
-        authenticated: true
-      }}
-    >
-    {}
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-}
-```
-
-If you don't want to specify all properties, you can either merge default values with overrides:
-
-```tsx
-const AppContext = createContext(appContextDefault);
-
-function App() {
-  return (
-    <AppContext.Provider
-      value={{
-        lang: "de",
-        ...appContextDefault
-      }}
-    >
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-}
-```
-
-Or you work without default values and use bind the generic type variable to bind context to a certain type:
-
-```tsx
-type AppContextValues = {
-  authenticated: boolean;
-  lang: string;
-  theme: string;
-}
-
-const AppContext = createContext<Partial<AppContextValues>>({});
-
-function App() {
-  return (
-    <AppContext.Provider
-      value={{
-        lang: "de"
-      }}
-    >
-      <ComponentThatUsesAppContext />
-    </AppContext.Provider>
-  );
-```
-
-All values become optional, so you have to do null checks when using them.
```

Comment:
This is explanation about `Partial` type.
I think it is unnecessary in Preact document.
Therefore, I removed this sentence.